databricks-catalog/main.tf

```diff
@@ -0,0 +1,52 @@
+resource "databricks_catalog" "catalog" {
```

Comment:
Given these are single bucket catalogs (we're not supporting prefixes), why split these two modules out? In case we do want to add prefix support later?

Reply:
I thought it'd be more straightforward if it were separated but I decided to create a list of maps so that it can all be passed in.